pkgs/top-level/python-packages.nix

```diff
@@ -27308,6 +27322,12 @@ EOF
 
   spacy = callPackage ../development/python-modules/spacy { };
 
+  spacy_models = callPackage ../development/python-modules/spacy/models.nix { };
```

Comment:
Might be nicer to add this to `passthru.models` of `spacy`. No need though.

Reply:
I think I will pass on this because the data model packages have dependencies on spacy.